addons/repair/report/repair_templates_repair_order.xml

```diff
@@ -10,14 +10,12 @@
                     <div t-if="o.partner_invoice_id">
                         <strong t-if="o.address_id != o.partner_invoice_id">Invoice address: </strong>
                         <div t-field="o.partner_invoice_id" 
-                            t-options='{"widget": "contact", "fields": ["address", "name", "phone"], "no_marker": True, "phone_icons": True}'/>
-                        <p t-if="o.partner_invoice_id.vat"><t t-esc="o.company_id.country_id.vat_label or 'Tax ID'"/>: <span t-field="o.partner_invoice_id.vat"/></p>
+                             t-options='{"widget": "contact", "fields": ["address", "name", "phone", "vat"], "no_marker": True, "phone_icons": True}'/>
                     </div>
                     <t t-if="o.address_id != o.partner_invoice_id">
                         <strong>Shipping address :</strong>
                         <div t-field="o.address_id" 
-                            t-options='{"widget": "contact", "fields": ["address", "name", "phone"], "no_marker": True, "phone_icons": True}'/>
```

Comment:
Do we really need VAT here?

Reply:
It has been added in this commit in a big refactor of the report RML -> Qweb (unless `display_address()` printed the VAT too, didn't go that far in the blame)

Reply:
And I don't think that a commit like this should change any behavior anyways